.github/vacuum-ruleset.yaml

```
@@ -602,3 +602,21 @@ rules:
         then:
             function: typedEnum
         type: validation
+    no-period-in-summary:
+        category:
+            id: operations
+            name: Operations
+        description: Path summary must not include period
+        formats:
+            - oas3
+            - oas3_1
+            - oas2
+        severity: error
+        given: $.paths[*][*].summary
```

🔧 Should we provide a how-to-fix hint as well?
```suggestion
        given: $.paths[*][*].summary
        howToFix: Summaries are short descriptions often used in navigation bars, they must not contain periods at the end. Please remove the period at the end of the summary text.
```